### PR TITLE
Change from Winston to PyPlot for plotting

### PR DESCRIPTION
**Pull request by obiajulu**
Switch in package for handling plotting.

### CODE REVIEW

**Comment by mauro3** [comment]
Not sure how far along this is, so some comments maybe unnecessary:
- remove any mention of Winston
- can the plotting.jl file not be updated to use PyPlot?
- plotting functions should not be specific to ODE.jl and neither their names

**Comment by obiajulu** [comment]
I made more changes and I think it are closer to being mergable.

File: examples/ex1.jl [hunk 1]
```diff
@@ -22,6 +22,7 @@ solver = S.allsolvers[[ODE.ode78,
 # make a TestRun which combines a TestCase with a Solver + some extras:
 ##Example protocol For adaptive solvers
 abstol = 1e-12
+
```
Comment (mauro3) [inline examples/ex1.jl]: Revert this. (Try not to have unrelated changes in your PRs)

File: src/plotting.jl [hunk 2]
```diff
@@ -12,13 +12,14 @@ for p in [:plot, :semilogx, :semilogy, :loglog, :oplot]
             ylabel(string(yfield)*" "*yunit)
         end
     end
+
```
Comment (mauro3) [inline src/plotting.jl:15]: winston still features in this file.  Do you need help dealing with this?

Edit: I dealt with it.

File: src/testcases/plei.jl [hunk 3]
```diff
@@ -46,6 +46,7 @@ plei = let
           for j =1:N
             if i != j
               r = ((y[i] -y[j])^2 + (y[i+N] -y[j+N])^2)^(3/2)
+
```
Comment (mauro3) [inline src/testcases/plei.jl]: remove change

**Comment by mauro3** [comment]
I tested it and did some extra changes. Maybe you can test and review it.  If happy merge.

**Comment by mauro3** [comment]
Cool, tnx @obiajulu!

File: testsuites/plot_suites.jl [hunk 4]
```diff
@@ -1,173 +1,127 @@
...
+        rsun = resSun[n]
+        for (s,res) in rsun
+            scd = getfield_vec(res, :scd)
+            if all(isnan(scd))
+                continue
+            end
+            maxscd = max(maxscd, maximum(scd))
+            wt = getfield_vec(res, :walltime)
+            @show cols[rem1(colind,nc)]
+            p = Py.plot(scd, wt, "-d"*cols[rem1(colind,nc)])
```
Comment (mauro3) [inline testsuites/plot_suites.jl:61]: Needs to be a semi-log-y plot.

File: testsuites/plot_suites.jl [hunk 5]
```diff
@@ -1,173 +1,127 @@
...
             continue
         end
-        maxscd = max(maxscd, maximum(scd))
-        wt = getfield_vec(res, :walltime)
-        if fst
-            p = W.semilogy(scd, wt, "o"*cols[colind])
-            fst = false
-        else
-            p = W.oplot(scd, wt, "o"*cols[rem1(colind,nc)])
+        fst = true
```
Comment (mauro3) [inline testsuites/plot_suites.jl:99]: What's this one for?
